docs/en_US/TrainingService/SupportTrainingService.md

```diff
@@ -0,0 +1,35 @@
+# Support TrainingService
+
+TrainingService is a concept of training platform that run trial jobs on the corresponding platform. NNI support [local](https://github.com/microsoft/nni/blob/master/docs/en_US/TrainingService/LocalMode.md), [remote](https://github.com/microsoft/nni/blob/master/docs/en_US/TrainingService/RemoteMachineMode.md), [pai](https://github.com/microsoft/nni/blob/master/docs/en_US/TrainingService/PaiMode.md), [kubeflow](https://github.com/microsoft/nni/blob/master/docs/en_US/TrainingService/KubeflowMode.md) and [frameworkcontroller](https://github.com/microsoft/nni/blob/master/docs/en_US/TrainingService/FrameworkControllerMode.md) training service. 
+
```

Comment:
NNI not only provides few built-in training service options, but also provides a method for customers to build their own training service easily.

Reply:
fixed.